product_docs/docs/pgd/5/upgrades/manual_overview.mdx

```diff
@@ -59,14 +59,19 @@ nodes are at the newer release. There must be no more than two versions of the
 software running at the same time. An upgrade must be completed, with all nodes
 fully upgraded, before starting another upgrade.
 
-An upgrade process can take more time when 
-caution is required to reduce business risk. However, we don't recommend
-running mixed versions of the software indefinitely.
-
-While you can use a rolling upgrade for upgrading a major version of the software,
-we don't support mixing PostgreSQL, EDB Postgres Extended, and
-EDB Postgres Advanced Server in one cluster. So you can't use this approach 
-to change the Postgres variant.
+An upgrade process can take more time when caution is required to reduce
```

Comment:
@djw-m update to match other language

Reply:
The other language is in a limitations section. This is a difference section and is already explicit in its recommendations.

Reply:
@djw-m it is explicit, but it's written in a way that makes it hard for the customer to understand the true impact. Recommend we explain more like suggested here https://github.com/EnterpriseDB/docs/pull/5725#discussion_r1626341673